dali/pipeline/operators/decoder/nvjpeg_decoder_test.cc

```diff
@@ -28,13 +28,18 @@ class nvjpegDecodeTest : public GenericDecoderTest<ImgType> {
       .AddOutput("decoded", "gpu");
```

Comment:
I would add TiffTestDecode method as well. Also Some test which decodes all files mixed together would be good.

Reply:
I'm going to add TiffTestDecode as soon as I can add some tiff test data. Otherwise, TiffTestDecode tests will fail, cause there will be no images to load

Reply:
Understood, so please remove all TIFF references from that patch.